jujugui/static/gui/src/app/components/isv-profile/plans-usage/plans-usage.js

```diff
@@ -0,0 +1,170 @@
+/*
+This file is part of the Juju GUI, which lets users view and manage Juju
+environments within a graphical interface (https://launchpad.net/juju-gui).
+Copyright (C) 2016 Canonical Ltd.
+
+This program is free software: you can redistribute it and/or modify it under
+the terms of the GNU Affero General Public License version 3, as published by
+the Free Software Foundation.
+
+This program is distributed in the hope that it will be useful, but WITHOUT
+ANY WARRANTY; without even the implied warranties of MERCHANTABILITY,
+SATISFACTORY QUALITY, or FITNESS FOR A PARTICULAR PURPOSE.  See the GNU Affero
+General Public License for more details.
+
+You should have received a copy of the GNU Affero General Public License along
+with this program.  If not, see <http://www.gnu.org/licenses/>.
+*/
+
+/*
+The ISV profile component renders a venders profile with stats based on there
+charms and bundles. You can only see this page is you have access.
+*/
+
+'use strict';
+
+YUI.add('plans-usage', function() {
+
+  juju.components.PlansUsage = React.createClass({
+    propTypes: {
+      d3: React.PropTypes.object.isRequired,
+      dataset: React.PropTypes.array.isRequired
+    },
+
+    componentDidMount: function() {
+      this.renderChart();
+      this.setState({d3Rendered: true});
+    },
+
+    shouldComponentUpdate: function shouldComponentUpdate() {
```

Comment:
you can remove the second shouldComponentUpdate unless you feel strongly about not having it be an anonymous fn.